data/org.freedesktop.portal.GlobalShortcuts.xml

```diff
@@ -0,0 +1,195 @@
+<?xml version="1.0"?>
+<!--
+ Copyright (C) 2022 Aleix Pol Gonzalez <aleixpol@kde.org>
+
+ This library is free software; you can redistribute it and/or
+ modify it under the terms of the GNU Lesser General Public
+ License as published by the Free Software Foundation; either
+ version 2 of the License, or (at your option) any later version.
+
+ This library is distributed in the hope that it will be useful,
+ but WITHOUT ANY WARRANTY; without even the implied warranty of
+ MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ Lesser General Public License for more details.
+
+ You should have received a copy of the GNU Lesser General Public
+ License along with this library. If not, see <http://www.gnu.org/licenses/>.
+
+ Author: Aleix Pol Gonzalez <aleixpol@kde.org>
+-->
+
+<node name="/" xmlns:doc="http://www.freedesktop.org/dbus/1.0/doc.dtd">
+  <!--
+      org.freedesktop.portal.GlobalShortcut:
+      @short_description: GlobalShortcut portal backend interface
+
+      This portal lets applications register global shortcuts. These shortcuts can be
+      used by apps to react to the user input, regardless of the input focus state.
+
+      To use this, an app has to create a session under which its actions
+      will be collected. Then shortcuts can be bound (see Bind), listed (see ListShortcuts).
+      The portal implementation performs the registration and tracks the shortcuts throughout
+      the session's lifespan.
+
+      When a shortcut triggers, Activated is emitted. When it is not triggered anymore, then
+      Deactivated is issued.
+  -->
+  <interface name="org.freedesktop.portal.GlobalShortcuts">
+      <!--
+        CreateSession:
+        @options: Vardict with optional further information
+        @handle: Object path for the #org.freedesktop.portal.Request object representing this call
+
+        Creates a global shortcuts session.
```

Comment:
This should describe what happens to the passed shortcuts; e.g. whether they are potentially restored from previous sessions. I assume they can't possibly be automatically bound the first time this is called for an app?

Reply:
Yes, it was indeed a bit weird. Massaged the wording across the document to make it all consistent.